### PR TITLE
Hook into new metrics views

### LINKED ISSUE #1118: Adaptation front métriques
On a rajouté des states : 

- Il s'est rien passé du tout
- Il s'est passé un truc dans l'année mais pas dans le mois
- Il s'est passé un truc dans l'année et dans le mois

Toutes les métriques sont en stock total mise à jour quotidiennement et le label exprime la variation sur le mois en cours (il n'y a plus qu'un label vert, pas de bleu ou de rouge). 

On affiche pas les sorties pour l'instant 
Par contre si on pourrait recup les fichiers ça serait cool :smile:
Voir les maquettes : https://www.figma.com/file/EgOgdOpYgOeS8UBh3tctSG/Dashboard-m%C3%A9triques?type=design&node-id=402%3A6153&mode=design&t=w3rxPiwcrKoDladW-1

### LINKED ISSUE #1121: Ajout de vues pour affichage des métriques sur le front

### LINKED ISSUE #1122: Job de mise à jour quotidienne des métriques dans mongo

### PR DESCRIPTION
**Pull request by maudetes**
Close https://github.com/etalab/data.gouv.fr/issues/1121
Close https://github.com/etalab/data.gouv.fr/issues/1122
Close https://github.com/etalab/data.gouv.fr/issues/1118

# Views

Moved logic from udata-front back here using hooks: https://github.com/etalab/udata-front/pull/260

We add metrics graph on dataset, reuse, organization and site dashboard pages and feed these with data from:
* our metrics API for traffic (views, download, external links)
* mongo for stock values (with an aggregation pipeline)

In both cases, we would like some caching features, to limit queries and pipelines being made.

We use metrics components defined in https://github.com/nicolaskempf57/udata-front/tree/feat/add-metrics-dashboard

To prevent conflicts, you'll need to use https://github.com/opendatateam/udata-piwik/pull/273 that removes legacy logic

# Metrics update job

Added a job to be scheduled to fetch total views for Dataset, Resource (Community included), Reuse and Organization objects.

# TODO
- [x] support resources downloads
- [x] ensure memoization works as expected (see https://github.com/opendatateam/udata-metrics/pull/8/commits/3be74ccdac5c1abcf0626baa8776ce3c800f9996)
- [x] improve metrics grid layout
- [x] support metric API not defined
- [x] improve variable name
- [x] discuss the possibility of having a daily job to compute those values (as udata-metrics did) -> yes
    - [x] implement it
    - [x] test it
    - [x] run it in dev on api-metrics : took **60m55.510s**
- [x] deal with global site metrics dashboard
- [x] add tests on stock metrics and metrics API parsing
- [x] add translations

### CODE REVIEW

**Review by geoffreyaldebert — APPROVED** [review]
👏

**Comment by nicolaskempf57** [comment]
There is now a `hide_chart` parameter on the `metrics` when the chart is not relevant.

**Review by quaxsze — APPROVED** [review]

File: requirements/install.pip [hunk 1]
```diff
@@ -1,2 +1,2 @@
-udata>=2.0.0
-influxdb==5.2.3
+udata>=6.0.0
+udata-front@git+https://github.com/maudetes/udata-front@feat/use-metrics-data
```
Comment (geoffreyaldebert) [inline requirements/install.pip]: maybe update that ? at least keep it in mind that it refers to a non-merged and published branch

File: .circleci/config.yml [hunk 2]
```diff
@@ -5,9 +5,8 @@ jobs:
   build:
     docker:
       - image: udata/circleci:2-alpine
-      - image: mongo:3.6
+      - image: mongo:6.0.4
```
Comment (geoffreyaldebert) [inline .circleci/config.yml:8]: readme may be updated with this change, with a section showing how to install it locally without pypi package

**Review by geoffreyaldebert — APPROVED** [review]
👏 I think I understood everything 🤞

File: .circleci/config.yml (hunk 2 above)
Comment (geoffreyaldebert) [reply .circleci/config.yml:8]: (yes this comment is not relative to this highlight line)

**Pull request merged**